lib/claide/informative_error.rb

```diff
@@ -12,7 +12,7 @@ module InformativeError
     # @return [Numeric] The exist status code that should be used to terminate
     #         the program with. Defaults to `1`.
     #
-    attr_accessor :exit_status
+    attr_writer :exit_status
```

Comment:
would this YARD comment fix the generated docs? I'm not sure how you're auto generating them.

```
# @!method exit_status
```